base_interface.go

```diff
@@ -70,6 +71,10 @@ func API(request rest.Request) (*rest.Response, error) {
 
 // MakeRequest attempts a Twilio SendGrid request synchronously.
 func MakeRequest(request rest.Request) (*rest.Response, error) {
+	if mock.Get() != nil {
```

Comment:
Mocking should happen within the test environment.

Reply:
greate, I think that would be better we use "startTest()" and "endTest()" for setting to begine and end test,
In this way, the developers doesn't depends to the name of environment variable for them project.